searchlib/src/vespa/searchlib/queryeval/andnotsearch.cpp

```diff
@@ -107,11 +121,16 @@ AndNotSearchStrict::internalSeek(uint32_t docid)
 
 std::unique_ptr<SearchIterator>
 AndNotSearch::create(ChildrenIterators children_in, bool strict) {
+    return create(std::move(children_in), false, strict);
+}
+
+std::unique_ptr<SearchIterator>
+AndNotSearch::create(ChildrenIterators children_in, bool elementwise, bool strict) {
```

Comment:
This seems a bit fragile. Consider either removing the old create function or using a typed flag.